src/config/graphical-session/wayland.md

```diff
@@ -85,9 +85,12 @@ compositors, and is installed as a dependency for most of them. Its package is
 ## Configuration
 
 The Wayland API uses the `XDG_RUNTIME_DIR` environment variable to determine the
-directory for the Wayland socket. In order to securely set this variable, you
-need the `elogind` service installed and enabled as your [session
-manager](./session-management.md).
+directory for the Wayland socket. You can either install `elogind` as your
+[session manager](./session-management.md), or you can manually set the
+environment variable.
```

Comment:
Throwing elogind here feels weird, making it clear that you can "install elogind as your session manager, which takes care of it, or..." seems better.